multinet/util.py

```diff
@@ -104,3 +104,8 @@ def data_path(file_name: str) -> str:
 def generate_arango_workspace_name() -> str:
     """Generate a string that can be used as an ArangoDB workspace name."""
     return f"w-{uuid1()}"
+
+
+def flask_secret_key() -> str:
+    """Load or create a flask secret key."""
+    return os.getenv("FLASK_SECRET_KEY") or uuid4().hex
```

Comment:
This function will return a different value each time it is called. Even if the function is only called once at startup, it would be better to secure it so it's idempotent. This is just to spare us future pain in case we ever do decide to call this function multiple times, not expecting it to return a random value each time.

I think I would do that by either calling `os.setenv()` (if such a thing exists), or by keeping a private variable here that you check for being `None` before setting it the first time.

Reply:
b047f7f